src/main/java/io/cryostat/net/web/http/api/v1/RecordingDeleteHandler.java

```diff
@@ -110,6 +117,13 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
                                 throw new HttpStatusException(500, e.getMessage(), e);
                             } finally {
                                 reportService.delete(recordingName);
+                                notificationFactory
+                                        .createBuilder()
+                                        .metaCategory(NOTIFICATION_CATEGORY)
+                                        .metaType(HttpMimeType.JSON)
+                                        .message(Map.of("recording", recordingName))
+                                        .build()
+                                        .send();
```

Comment:
I think this should be a few lines higher up in the main body of the `try` - if the deletion fails and an exception is thrown, then a notification should probably not be emitted.

Reply:
makes sense, I moved this and adjusted the test accordingly.